packages/node/src/backend.ts

```diff
@@ -30,10 +30,42 @@ export interface NodeOptions extends Options {
 
   /** HTTPS proxy certificates path */
   caCerts?: string;
+
+  /** Sets the number of context lines for each frame when loading a file. */
+  frameContextLines?: number;
 }
 
 /** The Sentry Node SDK Backend. */
 export class NodeBackend extends BaseBackend<NodeOptions> {
+  /**
+   * @inheritdoc
+   */
+  protected setupTransport(): Transport {
+    if (!this.options.dsn) {
+      // We return the noop transport here in case there is no Dsn.
+      return super.setupTransport();
+    }
+
+    const dsn = new Dsn(this.options.dsn);
+
+    const transportOptions = this.options.transportOptions ? this.options.transportOptions : { dsn };
+    const clientOptions = ['httpProxy', 'httpsProxy', 'caCerts'];
+
+    for (const option of clientOptions) {
+      if (this.options[option]) {
```

Comment:
This looks strange. "If we have `httpProxy` in the `options`, then assign `httpProxy` from `transportOptions`, otherwise use fallback from `options`". What if we don't have it in the `options` but we do in `transportOptions`?